publish.sh

```diff
@@ -16,20 +12,9 @@ test -s $userconfig && . $userconfig
 
 
 
-if [[ $userdoc ]] && [[ $userrn ]] && [[ $port ]] && [[ $server ]] && [[ $1 ]]; then
-  while [[ $1 ]]; do
-    if [[ $1 == 'doc' ]]; then
-      echo "Syncing documentation."
-      rsync $FILESDOC -e "ssh -p $port" ${userdoc}@${server}:doc.opensuse.org/htdocs/
-    elif [[ $1 == 'rn' ]]; then
-      echo "Syncing release notes."
-      cd release-notes
-      rsync $FILESRN  -e "ssh -p $port" ${userrn}@${server}:release-notes/
-    else
-      >&2 echo "Not sure how to interpret $1"
-    fi
-    shift
-  done
+if [[ $userdoc ]] && [[ $userrn ]] && [[ $port ]] && [[ $server ]]; then
```

Comment:
If we're not syncing anything into the `release-notes/` path anymore, there is no need to check for `userrn` anymore.